codex-rs/tui/src/chatwidget.rs

```diff
@@ -252,7 +280,7 @@ impl ChatWidget<'_> {
             }
             EventMsg::Error { message } => {
                 self.conversation_history
-                    .add_background_event(format!("Error: {message}"));
```

Comment:
If it is an error, then I think it should be presented as an error. Again, I think this makes the case for introducing a separate event type for interrupts.